apps/web/src/modules/job/appliedJob/components/JobCardContainer/components/JobCard/styled.ts

```diff
@@ -0,0 +1,14 @@
+import { styled } from '@mui/material'
+
+export const CardContainer = styled('div')`
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 0px;
```

Comment:
What's the purpose for this line ?

Reply:
should be some undeleted I test when try to style it 🤦‍♂️